### PR TITLE
Cloud liquid and ice number

### PR DESCRIPTION
**Pull request by pressel**
Adding number variables for cloud ice and liquid to Atmosphere.

### CODE REVIEW

File: haero/atmosphere.hpp [hunk 1]
```diff
@@ -51,9 +51,15 @@ class Atmosphere final {
   /// liquid water mass mixing ratio [kg vapor/kg dry air]
   ColumnView liquid_mixing_ratio;
 
+  /// grid box averaged cloud liquid number [#/kg dry air]
```
Comment (jeff-cohere) [inline haero/atmosphere.hpp:54]: is it worth calling this a number mixing ratio?

Comment (jeff-cohere) [reply haero/atmosphere.hpp:54]: (In the comment, I mean--the variable name is fine)

**Review by jeff-cohere — APPROVED** [review]
Looks good. A very minor comment.

EDIT: Looks like you have to modify `testing::create_atmosphere`.

File: haero/atmosphere.hpp (hunk 1 above)
Comment (pressel) [reply haero/atmosphere.hpp]: I actually went back and changed it in the variable name as well, it is long but useful.

**Pull request merged**